admin/load.php

```diff
@@ -156,17 +156,48 @@ function perflab_render_modules_page_field( $module_slug, $module_data, $module_
 				<input type="hidden" name="<?php echo esc_attr( "{$base_name}[enabled]" ); ?>" value="<?php echo $enabled ? '1' : '0'; ?>">
 				<?php
+				if ( 'database/sqlite' === $module_slug && file_exists( WP_CONTENT_DIR . '/db.php' ) && ! defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) ) {
+					printf(
+						/* translators: %s: WP_CONTENT_DIR */
+						esc_html__( 'The SQLite module cannot be activated because a different %s drop-in already exists.', 'performance-lab' ),
+						'<code>' . esc_html( WP_CONTENT_DIR ) . '/db.php</code>'
+					);
+				} elseif ( 'database/sqlite' === $module_slug && ! wp_is_writable( WP_CONTENT_DIR ) ) {
+					printf(
+						/* translators: %s: WP_CONTENT_DIR */
+						esc_html__( 'The SQLite module cannot be activated because the %s directory is not writable.', 'performance-lab' ),
+						'<code>' . esc_html( WP_CONTENT_DIR ) . '</code>'
+					);
+				} elseif ( 'database/sqlite' === $module_slug && ! class_exists( 'SQLite3' ) ) {
+					esc_html_e( 'The SQLite module cannot be activated because the SQLite extension is not loaded.', 'performance-lab' );
+				} else {
 					printf(
 						/* translators: %s: module name */
 						__( '%s is already part of your WordPress version and therefore cannot be loaded as part of the plugin.', 'performance-lab' ),
 						esc_html( $module_data['name'] )
 					);
+				}
 				?>
 			<?php } ?>
 		</label>
 		<p id="<?php echo esc_attr( "{$base_id}_description" ); ?>" class="description">
 			<?php echo esc_html( $module_data['description'] ); ?>
 		</p>
+		<?php if ( 'database/sqlite' === $module_slug ) : ?>
+			<?php if ( $enabled ) : ?>
+				<?php if ( defined( 'PERFLAB_SQLITE_DB_DROPIN_VERSION' ) ) : ?>
+					<?php // Don't use the WP notice classes here, as that makes them move to the top of the page. ?>
+					<p style="background:#fff;border:1px solid #c3c4c7;border-left-width: 4px;border-left-color:#dba617;box-shadow: 0 1px 1px rgba(0, 0, 0, 0.04);padding:1em;max-width:50em;">
+						<?php esc_html_e( 'Your site is currently using an SQLite database. You can disable this module to get back to your previous MySQL database, with all your previous data intact.', 'performance-lab' ); ?>
+					</p>
+				<?php endif; ?>
+			<?php else : ?>
+				<?php // Don't use the WP notice classes here, as that makes them move to the top of the page. ?>
+				<p style="background:#fff;border:1px solid #c3c4c7;border-left-width: 4px;border-left-color:#dba617;box-shadow: 0 1px 1px rgba(0, 0, 0, 0.04);padding:1em;max-width:50em;">
+					<?php esc_html_e( 'Enabling this module will switch to a separate database and install WordPress in it. You will need to reconfigure your site, and start with a fresh site. Disabling the module you will get back to your previous MySQL database, with all your previous data intact. Warning: Do not activate if your site is implemented in a multi-server environment.', 'performance-lab' ); ?>
+				</p>
```

Comment:
@aristath Default WP notice class working fine for me. In which case it will move to the top?
```suggestion
				<p class="notice notice-warning" style="padding:1em;max-width:50em;">
					<?php esc_html_e( 'Enabling this module will switch to a separate database and install WordPress in it. You will need to reconfigure your site, and start with a fresh site. Disabling the module you will get back to your previous MySQL database, with all your previous data intact. Warning: Do not activate if your site is implemented in a multi-server environment.', 'performance-lab' ); ?>
				</p>
```

Reply:
Hmmm I can no longer replicate... 🤔 
Seems to be working fine so let's go ahead and commit this 👍